src/utils/jdt.ts

```diff
@@ -0,0 +1,29 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license.
+
+import * as fse from "fs-extra";
+import * as path from "path";
+import { ProjectType } from "./webview";
```

Comment:
`utils/jdt.ts` requires `ProjectType` from `utils/webview.ts`. It looks strange. maybe ProjectType should also belong to jdt.ts?